### ISSUE TITLE
how to update fixed package versions from the `Add library +` button ?

### ISSUE THREAD

**Issue by 9il** [dlang-tour/core#729]
When a user adds the mir-algorithm library, `mir-algorithm 3.4.4` will be added.
Ideally is to replace them with `*`.

**Comment by ZombineDev** [comment]
The list of installed packages in the docker image is obtained dynamically here: https://github.com/dlang-tour/core/blob/9014e140e5006a6e58400924c8a4729a5cd43db7/source/exec/docker.d#L190.
Everytime a new Docker image is built we fetch the newest available packages.
Showing arbitrary versions however is not a good idea, since the user will expect them to work, while only a version per package is actually installed.

**Comment by 9il** [comment]
Agreed. I mean "Add library" button at run.dlang.io. It has a fixed package version. Instead, it would be more sensitive to add `"*"`. I couldn't find where `3.4.4` is hardcoded.

![image](https://user-images.githubusercontent.com/845083/60765333-54021400-a0c3-11e9-8915-9a82a3f7d36b.png)

**Comment by wilzbach** [comment]
Here:

https://github.com/dlang-tour/core/blob/9014e140e5006a6e58400924c8a4729a5cd43db7/public/static/js/tour-controller.js#L409
